docs/kh2/file/type/00battle.md

```diff
@@ -39,46 +41,96 @@ The damage effects' values on MSET files point to this table.
 
 | Offset | Variable Type | Description |
 |--------|---------------|-------------|
-| 0 	 | uint | File type (6)
-| 4 	 | uint | Entry Count
+| 0 	 | uint32 | File type (6)
+| 4 	 | uint32 | Entry Count
 
 ### Atkp Entry
 
 | Offset | Variable Type | Description |
 |--------|---------------|-------------|
-| 0 	 | ushort | SubId
```

Comment:
Is there a reason the offset for some of the tables is decimal and others hexadecimal?

Reply:
Different contributors